### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by arbitur**
No need to add another tag since no actual code changes was made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+## Installation 
+
+### Cocoapods
+	
+	pod 'Personnummer', '~> 1.0.0'
+	
+## Usage
+
+    // Validate
```
Comment (frozzare) [inline README.md]: Wrap this in syntax highlighting blocks https://help.github.com/articles/creating-and-highlighting-code-blocks/

I think it should be 
<pre>```swift
code
```</pre>

File: README.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
-WIP: Validate Swedish social security numbers
+# Personnummer swift
+
+Small library to validate and format swedish social security numbers called "Personnummer".
+
+## Installation 
+
+### Cocoapods
+	
+	pod 'Personnummer', '~> 1.0.0'
```
Comment (frozzare) [inline README.md]: See comment below.

**Comment by arbitur** [comment]
It is done.

**Comment by frozzare** [comment]
Looks good!

**Pull request merged**